samples/word/99-preview-apis/manage-list-styles.yaml

```diff
@@ -0,0 +1,112 @@
+id: word-lists-manage-styles
+name: Manage list styles
+description: This sample shows how to get the list styles in the current document.
+author: xiruatms
+host: WORD
+api_set:
+    WordApi: '1.9'
+script:
+    content: |
+        $("#count").on("click", () => tryCatch(getCount));
+        $("#properties").on("click", () => tryCatch(getProperties));
+
+        async function getCount() {
+          // Gets the available list styles stored with the document.
+          await Word.run(async (context) => {
+            const styles: Word.StyleCollection = context.document.getStyles();
+            const count = styles.getCount();
+            styles.load();
+            await context.sync();
+
+            for (let i = 0; i <= count.value; i++) {
+              if (styles.items[i] && styles.items[i].type == "List") {
+                console.log(`List style name: ${styles.items[i].nameLocal}`, styles.items[i]);
+              }
+            }
+          });
+        }
+
+        async function getProperties() {
```

Comment:
The only properties shown are the list levels. Can we rename this function (and the buttons and comments) to reflect that?

Reply:
Right now, we're logging all the properties of the list style along with the list levels so the name, etc. is appropriate.